doc/radosgw/notifications.rst

```diff
@@ -70,6 +71,8 @@ which tells the client that it may retry later.
 
 .. note:: If the notification fails with an error, cannot be delivered, or
    times out, it is retried until it is successfully acknowledged.
+   You can control its retry with ``time_to_live``/``max_retries`` to have a time/retry limit and
```

Comment:
i dont think we have these in "reef".
when making language and style fixes to docs, please make sure that you do not change anything that describe functionality and product behavior.